cypress/support/api_commands.js

```diff
@@ -104,6 +104,26 @@ Cypress.Commands.add('apiDeleteChannel', (channelId) => {
     });
 });
 
+/**
+ * Updates a channel directly via API
+ * This API assume that the user is logged in and has cookie to access
+ * @param {String} channelId - The channel's id, not updatable
+ * @param {String} name - The unique handle for the channel, will be present in the channel URL
+ * @param {String} displayName - The non-unique UI name for the channel
+ * @param {String} type - 'O' for a public channel (default), 'P' for a private channel
+ * @param {String} purpose - A short description of the purpose of the channel
+ * @param {String} header - Markdown-formatted text to display in the header of the channel
+ * Only channelId is required
+ */
+Cypress.Commands.add('apiUpdateChannel', (parameters) => {
```

Comment:
@thekiiingbob Any suggestions for a better parameter name? I would have enumerated each property but only the channelId is required and I didn't want the function call to look like `apiUpdateChannel(channelId, , , , , 'Changing header')`

Reply:
I suggest `channel`.

Reply:
Thanks!